packs/data/pathfinder-bestiary-3.db/android-infiltrator.json

```
@@ -764,7 +846,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The android infiltrator deals an additional 1d6 precision damage to flat-footed creatures.</p>"
+                    "value": "<p>The android infiltrator deals an additional [[/r {1d6}[precision]]]{1d6 precision damage} to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```

```suggestion
                    "value": "<p>The android infiltrator deals an additional 1d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
I think we've settled on not making these rollable since the damage type is wrong anyways and the automation is pretty solid.

---

Oh, good to know, I'll keep that in mind for further reviews.